camelot/view/action_steps/item_view.py

```diff
@@ -88,6 +93,19 @@ class UpdateTableView( ActionStep ):
     
     """
 
+    #value: not needed
+    search_text: typing.Union[str, None]
+    title: typing.Union[str, ugettext_lazy]
+    #filters: TODO
+    #list_actions: TODO
+    #columns: TODO
+    #left_toolbar_actions: TODO
+    #right_toolbar_actions: TODO
+    #top_toolbar_actions: TODO
+    #bottom_toolbar_actions: TODO
+    #list_action: TODO
+    proxy: int # id of proxy in ProxyRegistry
```

Comment:
for the sake of the 'protocol definition' tss server en client zou ik hier durven van maken : 

proxy_route:  Route

zodat alle client side pointers naar server side objecten dezelfde structuur volgen, die route zelf
kan dan een eenvoudige ['123'] zijn ...

Reply:
done